packages/lesswrong/components/posts/PostsGroupDetails.tsx

```diff
@@ -25,7 +25,7 @@ const styles = (theme: ThemeType): JssStyles => ({
     fontFamily: theme.typography.fontFamily
   },
   root: {
-    marginBottom: 5, 
+    marginBottom: 12, 
```

Comment:
forum-gate?

Reply:
I checked with them about this one.

Reply:
oh, I may not have specifically asked about this one. Gated now.